tests/runtime/test_command_buffer.c

```diff
@@ -0,0 +1,186 @@
+#include <stdio.h>
```

Comment:
Who has the copyrights of this file? Was the test case copied from somewhere?

Reply:
This is adapted from the sample code in the spec, not entirely sure how that usually goes.